lib/pageflow/engine.rb

```diff
@@ -88,5 +69,27 @@ class Engine < ::Rails::Engine
         FactoryGirl.definition_file_paths.unshift(Engine.root.join('spec', 'factories'))
       end
     end
+
+    # Precompile additional assets. pageflow/editor.* has to be
+    # provided by the main app.
+    initializer 'pageflow.assets.precompile' do |app|
+      app.config.assets.precompile += %w(pageflow/editor.js pageflow/editor.css
+                                     pageflow/application_with_simulated_media_queries.css
+                                     pageflow/print_view.css
+                                     pageflow/lt_ie9.js pageflow/lt_ie9.css pageflow/ie9.js pageflow/ie9.css
+                                     pageflow/vendor.js
+                                     video-js.swf vjs.eot vjs.svg vjs.ttf vjs.woff)
+
+       app.config.assets.precompile << lambda do |path, _filename|
+         Pageflow.config.themes.any? do |theme|
+           path == theme.stylesheet_path
+         end
+       end
+
+       app.config.assets.precompile << lambda do |path, filename|
+         filename.start_with?(Engine.root.join('app/assets').to_s) &&
+           !['.js', '.css', ''].include?(File.extname(path))
+       end
+    end
```

Comment:
What difference does wrapping the code in an `initializer` block make here? Can we fix indentation here?

Reply:
Just following the documented API for engine asset precompilation. Pretty sure this will handle a tweaked asset pipeline better.

Indentation fixed.